app/controllers/request_controller.rb

```diff
@@ -781,7 +781,7 @@ def upload_response
       flash[:notice] = _("Thank you for responding to this FOI request! " \
                            "Your response has been published below, and a " \
                            "link to your response has been emailed to {{user_name}}.",
-                         :user_name => CGI.escapeHTML(@info_request.user.name))
+                         :user_name => @info_request.user.name.html_safe)
```

Comment:
yeah, I inadvertently check that it **had** escaped the value rather than that it hadn't